orange/README.md

```diff
@@ -270,6 +270,7 @@ investigate potential causes for QC failure.
     - Add potentially interesting chromosomal rearrangements (1q trisomy and 1p19q co-deletion) to report
     - Derive breakend fields type, chromosome, chromosomeBand, orientation and junctionCopyNumber from root sources
     - Make PurpleGeneCopyNumber transcript-aware in orange-datamodel
+    - Upgrade DOID-json to Dec 2024 release  
```

Comment:
Minor but I would rephrase to "Upgraded DOID datamodel to version of Dec 2024 release". 

The actual DOID json is not part of the code in the end, and fair chance this datamodel works with a next release as well since the datamodel doesn't change too often.